dev/dll/Microsoft.UI.Xaml.vcxproj.filters

```diff
@@ -147,225 +147,6 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
```

Comment:
how many of these do we need ?

Reply:
I'm not sure.. these are just the ones added my that previous PR, I just reverted the file to the state before that check in

Reply:
@llongley fixed this a while back, it might have regressed.